development/bootstrap-job-preparation/bootstrap-job.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+if [ $# -ne 3 ]; then
+  echo "Usage: $0 <bootstrap_job_version> <repo_username> <repo_api_key>"
+  exit 1
+fi
+
+BOOTSTRAP_JOB_VERSION=$1
+export ARTIFACTS_CREDS_USR=$2
+export ARTIFACTS_CREDS_PSW=$3
+
+mvn clean process-resources -Dbootstrap.job.version="${BOOTSTRAP_JOB_VERSION}" -s settings.xml
+
+cd target/bootstrap-job
+
+mvn clean install -s ../../settings.xml
+
+docker run -d -p 8000:5000 --name local-registry registry:2
```

Comment:
what's the purpose of starting local registry, especially that the image can be built directly to the docker daemon?

Reply:
No purpose, it appears that when I was using `local-client` my command got messy. after your advice, it is working. thanks man.